### PR TITLE
 API Dev updates

### PR DESCRIPTION
**Pull request by ArifPMan**

### CODE REVIEW

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 1]
```diff
@@ -60,109 +63,137 @@ Languages supported:
...
 
 In the above screen, select a corresponding collection from the list and click **Add Contract Test**. The contract tests attached to the collection are now added to the API. Only available collections will be shown in this list.
 
-### Observing an API
+## Observing an API
 
-You can link your monitors running on collections in your current workspace with an API. 
+You can link your monitors running on collections in your current workspace with an API.
 
 In the **Observe** tab, click **Add a monitor**. The following screen opens:
```
Comment (sankalp0o) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: @ArifPMan We also need a new section for "Creating a monitor" on the same lines as "Creating a mock server"

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 2]
```diff
@@ -60,109 +63,137 @@ Languages supported:
...
+[![add existing mock](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Mock-1.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Mock-1.png)
+
+In the above screen, select **Create new mock server**. The following screen appears:
 
-You can link mock servers running on a collection to an API. In the **Develop** tab, click **Add a mock server**. The following screen opens:
+[![add existing mock](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Mock-2.png)](https://s3.amazonaws.com/postman-static-getpostman-com/postman-docs/API-Mock-2.png)
+
+Click **Generate collection from schema** to create a mock server on the collection you generated from your schema. Specify a name in **Name your collection** field and click **Generate collection and continue**. 
+
+From the **Develop** tab, click **Add Mock Server**.
```
Comment (sankalp0o) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: Not sure why, but the line "From the Develop tab, click Add Mock Server " is repeated 4 times.

Comment (sankalp0o) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: @ArifPMan ^ Was this corrected?

File: v6/postman/design_and_develop_apis/the_api_workflow.md [hunk 3]
```diff
@@ -181,10 +181,42 @@ In the above screen, select a corresponding collection from the list and click *
...
+
+
+#### Creating new monitor
+
+From the **Observe** tab, click **Create new Monitor**. The following screen opens:
+
+
+Create new monitor screen.
+
+Under **Select requests to monitor** tab, specify the name of the collection on which you want to create a monitor.
```
Comment (sankalp0o) [inline v6/postman/design_and_develop_apis/the_api_workflow.md]: This is not a tab, this is a step

File: v6/postman/design_and_develop_apis/the_api_workflow.md (hunk 2 above)
Comment (ArifPMan) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: @sankalp0o I can't reproduce this issue in my environment. Will double check again.

File: v6/postman/design_and_develop_apis/the_api_workflow.md (hunk 3 above)
Comment (ArifPMan) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: This is addressed.

File: v6/postman/design_and_develop_apis/the_api_workflow.md (hunk 2 above)
Comment (sankalp0o) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: @ArifPMan I can see this both in code and in my local build.

Comment (ArifPMan) [reply v6/postman/design_and_develop_apis/the_api_workflow.md]: @sankalp0o  - Please check now and let me know.

**Pull request merged**